src/components/directorybrowser/directorybrowser.js

```diff
@@ -1,51 +1,148 @@
+                html += Globalize.translate("ButtonMoreInformation");
+                html += "</a>";
+            } else if("linux" === systemInfo.OperatingSystem.toLowerCase()) {
+                html += "<br/>";
+                html += "<br/>";
+                html += Globalize.translate("MessageDirectoryPickerLinuxInstruction");
+                html += "<br/>";
+            }
+            html += "</div>"
         }
-        html += '<form style="margin:auto;">', html += '<div class="inputContainer" style="display: flex; align-items: center;">', html += '<div style="flex-grow:1;">';
-        var labelKey = !0 !== options.includeFiles ? "LabelFolder" : "LabelPath",
-            readOnlyAttribute = options.pathReadOnly ? " readonly" : "";
-        return html += '<input is="emby-input" id="txtDirectoryPickerPath" type="text" required="required" ' + readOnlyAttribute + ' label="' + Globalize.translate(labelKey) + '"/>', html += "</div>", readOnlyAttribute || (html += '<button type="button" is="paper-icon-button-light" class="btnRefreshDirectories emby-input-iconbutton" title="' + Globalize.translate("ButtonRefresh") + '"><i class="md-icon">search</i></button>'), html += "</div>", readOnlyAttribute || (html += '<div class="results paperList" style="max-height: 200px; overflow-y: auto;"></div>'), options.enableNetworkSharePath && (html += '<div class="inputContainer" style="margin-top:2em;">', html += '<input is="emby-input" id="txtNetworkPath" type="text" label="' + Globalize.translate("LabelOptionalNetworkPath") + '"/>', html += '<div class="fieldDescription">', html += Globalize.translate("LabelOptionalNetworkPathHelp"), html += "</div>", html += "</div>"), html += '<div class="formDialogFooter">', html += '<button is="emby-button" type="submit" class="raised button-submit block formDialogFooterItem">' + Globalize.translate("ButtonOk") + "</button>", html += "</div>", html += "</form>", html += "</div>", html += "</div>", html += "</div>"
+        html += '<form style="margin:auto;">';
+        html += '<div class="inputContainer" style="display: flex; align-items: center;">';
+        html += '<div style="flex-grow:1;">';
+        var labelKey;
+        if(options.includeFiles !== true) {
+            labelKey = "LabelFolder";
+        } else {
+            labelKey = "LabelPath";
+        }
+        var readOnlyAttribute = options.pathReadOnly ? " readonly" : "";
+        html += '<input is="emby-input" id="txtDirectoryPickerPath" type="text" required="required" ' + readOnlyAttribute + ' label="' + Globalize.translate(labelKey) + '"/>';
+        html += "</div>";
+        readOnlyAttribute || (html += '<button type="button" is="paper-icon-button-light" class="btnRefreshDirectories emby-input-iconbutton" title="' + Globalize.translate("ButtonRefresh") + '"><i class="md-icon">search</i></button>');
```

Comment:
Let's de-uglify this piece, too:
```suggestion
        if (!readOnlyAttribute) {
            html += '<button type="button" is="paper-icon-button-light" class="btnRefreshDirectories emby-input-iconbutton" title="' + Globalize.translate("ButtonRefresh") + '">';
            html += '<i class="md-icon">search</i>';
            html += '</button>');
        }
```